src/main/java/organice/logic/parser/ProcessingCommandParser.java

```diff
@@ -0,0 +1,38 @@
+package organice.logic.parser;
+
+import static organice.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static java.util.Objects.requireNonNull;
+import static organice.logic.parser.CliSyntax.PREFIX_NRIC;
+
+import java.util.Arrays;
+
+import organice.logic.commands.ProcessingCommand;
+import organice.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FindCommand object
+ */
+public class ProcessingCommandParser implements Parser<ProcessingCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ProcessingCommand
+     * and returns a ProcessingCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ProcessingCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        String trimmedArgs = args.trim();
+        String firstNRIC = null;
```

Comment:
Not a good practice to initialise with null values. You can do something like this:
```
String firstNric;
String secondNric;
```
And leave it to the later parts of the code will initialise `firstNric` and `secondNric`.

Reply:
amended